packages/@react-spectrum/s2/src/Tabs.tsx

```diff
@@ -10,9 +10,289 @@
  * governing permissions and limitations under the License.
  */
 
-import {Tabs as RACTabs, TabsProps} from 'react-aria-components';
+import {TabListProps as AriaTabListProps, TabPanel as AriaTabPanel, TabPanelProps as AriaTabPanelProps, TabProps as AriaTabProps, TabsProps as AriaTabsProps, Provider, Tab as RACTab, TabList as RACTabList, Tabs as RACTabs, TabListStateContext} from 'react-aria-components';
+import {Collection, DOMRef, Key, Orientation} from '@react-types/shared';
+import {createContext, forwardRef, ReactNode, useCallback, useContext, useEffect, useRef, useState} from 'react';
+import {focusRing, getAllowedOverrides, StyleProps} from './style-utils' with {type: 'macro'};
+import {IconContext} from './Icon';
+import {raw} from '../style/style-macro' with {type: 'macro'};
+import {size, style} from '../style/spectrum-theme' with {type: 'macro'};
+import {useDOMRef} from '@react-spectrum/utils';
+import {useLayoutEffect} from '@react-aria/utils';
+import {useLocale} from '@react-aria/i18n';
 
+interface TabsProps extends Omit<AriaTabsProps, 'className' | 'style' | 'children'>, StyleProps { // is height a style prop we want to allow for tabs?
+  children?: ReactNode,
+  density?: 'compact' | 'regular'
+}
+
+interface TabProps extends Omit<AriaTabProps, 'children' | 'style' | 'className'> {
+  children?: ReactNode
+}
+
+interface TabListProps<T> extends Omit<AriaTabListProps<T>, 'children' | 'style' | 'className'>{
+  children?: ReactNode
+}
+
+interface TabPanelProps extends Omit<AriaTabPanelProps, 'children' | 'style' | 'className'>{
+  children?: ReactNode
+}
+
+export function TabPanel(props: TabPanelProps) {
+  return (
+    <AriaTabPanel {...props} className={style({marginTop: 4, color: 'neutral'})} />
+  );
+}
+
+const tab = style({
+  ...focusRing(),
+  display: 'inline-flex',
+  color: {
+    default: 'neutral-subdued',
+    isSelected: 'neutral',
+    isHovered: 'neutral-subdued',
+    isDisabled: 'disabled'
```

Comment:
don't forget HCM

Reply:
ah right, i always forget. thanks for reminding me